### PR TITLE
Ria 1304 authenticate home office user

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-1304](https://tools.hmcts.net/jira/browse/RIA-1304)

### Change description ###

See Jira

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/features/stepDefinitions/authentication.steps.ts [hunk 1]
```diff
@@ -69,6 +85,50 @@ Given(/^I switch to be a `?(?:Solicitor|Legal Rep)(?:| A)`?$/, async function ()
     await ccdPage.contentContains('Immigration');
 });
 
+Given(/^I switch to be a `?(?:Home Office APC)`?$/, async function () {
+    await browser.sleep(100);
```
Comment (jkumar-moj) [inline e2e/features/stepDefinitions/authentication.steps.ts]: Seems like lot of duplication here except the signIn part. Could we generify in anyway?

File: e2e/flows/authentication.flow.ts [hunk 2]
```diff
@@ -31,6 +31,50 @@ export class AuthenticationFlow {
         await this.waitForAngularIfRequired();
     }
 
+    async signInAsHomeOfficeApc() {
+        await this.signOut();
+        await this.idamSignInPage.waitUntilLoaded();
```
Comment (jkumar-moj) [inline e2e/flows/authentication.flow.ts]: I am thinking we could do refactoring similar to the above here as well.

File: e2e/features/stepDefinitions/authentication.steps.ts (hunk 1 above)
Comment (bradsorour) [reply e2e/features/stepDefinitions/authentication.steps.ts]: Specific to each user as per the existing code base. We could look to refactor at some point in the future.

File: e2e/flows/authentication.flow.ts (hunk 2 above)
Comment (bradsorour) [reply e2e/flows/authentication.flow.ts]: As per the existing code base. Could consider rebasing in the future.

**Pull request merged**